---
layout: page
title: Na Jiao
position: Principle Investigator
image: /people/NaJiao.png
degree: Ph.D.
email: jiaonall@163.com
webside: https://jiaonall.github.io/
github: https://github.com/jiaonall
google_scholar: OqbAECkAAAAJ
office: Zhejiang University, xxx Road, HangZhou, Zhejiang, China
introduce: <b>Na Jiao</b> is a young researcher with B.S. and Ph.D. in Bioinformatics from Tongji University. Now she is a Professor at the National Clinical Research Center for Child Health, the Children’s Hospital, Zhejiang University School of Medicine. Her research focuses on <b>Host-Microbiome Crosstalk</b>, <b>Heterogeneous (omics-)data integration</b>, <b>Biomarkers</b>, and <b>Artificial intelligence</b>.
---
<style>
p {
    text-align: justify;
}
</style>

<h2>Na Jiao <small>Professor</small></h2>
<hr>

<div class='row'>
  <div class="col-lg-8">
    <p><b>Education:</b> Ph.D.</p>
    <p><b>Research areas:</b> Bioinformatics & Metagenomics </p>
    <p><b>Research focus:</b> Host-Microbiome Crosstalk, Heterogeneous (omics-)data integration, Biomarkers and Artificial intelligence</p>
  </div>
  <div class="col-lg-4">
    <div class="text-right">
        <img alt="najiao" src="/assets/img/people/NaJiao.png" width="200px" />
    </div>
  </div>
</div>

**Current position：**Distinguished research fellow (Professor), Children's Hospital of Zhejiang University School of Medicine

**More about me：**[焦娜-浙江大学个人主页](https://person.zju.edu.cn/jiaona)、[焦娜-同济CADD课题组个人主页](https://cadd.tongji.edu.cn/info/1053/1143.htm)

**Work Experience**

* Sep. 2021-Present: Distinguished research fellow (Professor), National Clinical Research Center for Child Health, Children's Hospital of Zhejiang University School of Medicine

* Sep. 2019-Aug. 2021: Postdoctoral fellow, Bioinformatics, The Sixth Affiliated Hospital of Sun Yat-sen University, Guangzhou, China

**Education**
* **Ph.D.**: Jun. 2019, Bioinformatics, School of Life Sciences and Technology, Tongji University, Shanghai, China


* **B.S.**: Jun. 2014, Food Science and Engineering, College of Biological and Agricultural Engineering, Jilin University, Jilin, China

**Research Project**

1. National Natural Science Foundation of China, Young Scientists Fund, 82000536, Mechanism for the gut microbiota-mediated bile acid metabolism promoting NAFLD pathogenesis via gut-liver axis, 2021-01 to 2023-12, 240000, in research，PI

2. Higher Education Institutions Fundamental Research Project, Young Teachers Training Programs, 20ykpy07, Altered gut microbiome and mechanism in FUT2 loss-of-function mutants in inflammatory bowel disease, 2020-01to 2021-01, 120000, conclusion，PI

3. China Postdoctoral Science Foundation, General Program, 2019M663252, Role of the choline metabolizing microbiota in the pathogenesis of non-alcoholic fatty liver disease, 2020-01to 2021-08，80000, conclusion，PI

4. China Postdoctoral Science Foundation, Postdoctoral Innovative Talent Support Program, BX20190393, Identification of bile acid metabolizing bacteria and individualized intervention in the pathogenesis of nonalcoholic fatty liver disease, 2019-09 to 2021-08, 600000, conclusion，PI

**Publications**

1. Lei Liu<sup>†</sup>, Ruixin Zhu,and **Dingfeng Wu**<sup>\*</sup>. Misuse of reporter score in microbial enrichment analysis. **iMeta**. **2023**; e95.https://doi.org/10.1002/imt2.95

1. **Dingfeng Wu**<sup>†</sup>, Lei Liu<sup>†</sup>, Na Jiao<sup>†</sup>, Yida Zhang, Li Yang, Chuan Tian, Ping Lan, Lixin Zhu<sup>\*</sup>, Rohit Loomba<sup>\*</sup>, and Ruixin Zhu<sup>\*</sup>. Targeting Keystone Species Helps Restore the Dysbiosis of Butyrate-Producing Bacteria in Nonalcoholic Fatty Liver Disease. **iMeta**, **2022**; e61. https://doi.org/10.1002/imt2.61

1.  Jing Zhong<sup>†</sup>, **Dingfeng Wu**<sup>†</sup>, Yuanyuan Zeng<sup>†</sup>, Gaosong Wu<sup>†</sup>, Ningning Zheng, Wenjin Huang, Yan Li, Xin Tao, Weize Zhu, Lili Sheng, Xiaoxu Shen, Weidong Zhang<sup>\*</sup>, Ruixin Zhu<sup>\*</sup>, and Houkai Li<sup>\*</sup>. The Microbial and Metabolic Signatures of Patients with Stable Coronary Artery Disease. **Microbiology spectrum**, **2022**; 10(6):e0246722. doi:10.1128/spectrum.02467-22

1. Sijing Cheng<sup>†</sup>, **Dingfeng Wu**<sup>†</sup>, Jie Li, Yifeng Zou, Yunle Wan, Lihan Shen, Lixin Zhu<sup>\*</sup>, Mang Shi, Linlin Hou, Tao Xu, Na Jiao, Yichen Li, Yibo Huang, Zhipeng Tang, Mingwei Xu, Shusong Jiang, Maokun Li<sup>\*</sup>, Guangjun Yan<sup>\*</sup>, Ping Lan<sup>\*</sup>, and Ruixin Zhu<sup>\*</sup>. Risk factors for the critical illness in SARS-CoV-2 infection: a multicenter retrospective cohort study. **Respiratory Research**, **2020**; 21(1), 277. DOI: https://doi.org/10.1186/s12931-020-01492-zLu.

1. Fan<sup>†,\*</sup>, **Dingfeng Wu**<sup>†</sup>, Vadim Goremykin, Jing Xiao, YanbingXu, Sriram Garg, Chuanlun Zhang, William F. Martin<sup>\*</sup>, RuixinZhu<sup>\*</sup>. Phylogenetic analyses with systematic taxon sampling show that mitochondria branch within Alphaproteobacteria. **Nature Ecology & Evolution**, **2020**; 4(9), 1213–1219. DOI: https://doi.org/10.1038/s41559-020-1239-x.  

1. **Dingfeng Wu**<sup>†</sup>, WenxingGao<sup>†</sup>, Xiaoyi Li<sup>†</sup>, Chuan Tian, Na Jiao, Sa Fang, Jing Xiao, Zhifeng Xu, Lixin Zhu<sup>\*</sup>, Guoqing Zhang<sup>\*</sup> and Ruixin Zhu<sup>\*</sup>. Dr AFC: Drug Repositioning Through Anti-Fibrosis Characteristic. **Briefings in Bioinformatics**, **2020**; 22(3). DOI: https://doi.org/10.1093/bib/bbaa115.

1. Yunle Wan<sup>†</sup>, Jie Li<sup>†</sup>, Lihan Shen<sup>†</sup>,Yifeng Zou<sup>†</sup>, Linlin Hou<sup>†</sup>, Lixin Zhu<sup>†,\*</sup>, Howard S.Faden, Zhipeng Tang, Mang Shi, Na Jiao, Yichen Li, Sijing Cheng, Yibo Huang, **Dingfeng Wu**, ZhifengXu, Linnuo Pan, Jun Zhu, GuangjunYan<sup>\*</sup>, Ruixin Zhu<sup>\*</sup>, and Ping Lan<sup>\*</sup>. Enteric involvement in hospitalised patients with COVID-19 outside Wuhan. **The Lancet Gastroenterology & Hepatology**, **2020**; 5(6), 534–535. DOI: https://doi.org/10.1016/S2468-1253(20)30118-7.

1. Tianyi Qiu<sup>†</sup>, Yiyan Yang<sup>†</sup>, JingxuanQiu, Yang Huang, Tianlei Xu, Han Xiao, **Dingfeng Wu**, QingchenZhang, Chen Zhou, Xiaoyan Zhang, KailinTang, Jianqing Xu<sup>\*</sup>, and ZhiweiCao<sup>\*</sup>. CE-BLAST makes it possible to compute antigenic similarity for newly emerging pathogens. **Nature communications**, **2018**; 9(1), 1772. DOI: https://doi.org/10.1038/s41467-018-04171-2.

1. Tianyi Qiu<sup>†</sup>, **Dingfeng Wu**<sup>†</sup>, JingxuanQiu, and Zhiwei Cao<sup>\*</sup>. Finding the molecular scaffold of nuclear receptor inhibitors through high-throughput screening based on Proteochemometric modelling. **Journal of cheminformatics**, **2018**; 10(1), 21. DOI: https://doi.org/10.1186/s13321-018-0275-x.

1. Tianyi Qiu<sup>†</sup>, **Dingfeng Wu**<sup>†</sup>, LinLin Yang, Hao Ye, Qiming Wang, Zhiwei Cao<sup>\*</sup>, and Kailin Tang<sup>\*</sup>. Exploring the mechanism of flavonoids through systematic bioinformatics analysis. **Frontiers in pharmacology**, **2018**; 9:918. DOI: https://doi.org/10.3389/fphar.2018.00918.

1. Tianyi Qiu<sup>†</sup>, Jingxuan Qiu, Jun Feng, **Dingfeng Wu**, YiyanYang, Kailin Tang, ZhiweiCao, and Ruixin Zhu<sup>\*</sup>. The recent progress in proteochemometricmodelling: focusing on target descriptors, cross-term descriptors and application scope. **Briefings in bioinformatics**, **2016**. 18(1), 125–136. DOI: https://doi.org/10.1093/bib/bbw004.

1. Jingxuan Qiu<sup>†</sup>,Tianyi Qiu, Yiyan Yang, **Dingfeng Wu**, and Zhiwei Cao<sup>\*</sup>. Incorporating structure context of HA protein to improve antigenicity calculation for influenza virus A/H3N2. **Scientific reports**, **2016**; 6, 31156. DOI: https://doi.org/10.1038/srep31156.

1. Jun Gao<sup>†</sup>, QingchenZhang<sup>†</sup>, Min Liu, Lixin Zhu, **Dingfeng Wu**, Zhiwei Cao, and RuixinZhu<sup>\*</sup>. bSiteFinder, an improved protein-binding sites prediction server based on structural alignment: more accurate and less time-consuming. **Journal of cheminformatics**, **2016**; 8, 38. DOI: https://doi.org/10.1186/s13321-016-0149-z.

1. Lei Liu<sup>†</sup>, Maria Tsompana, Yong Wang, **Dingfeng Wu**, Lixin Zhu, and RuixinZhu<sup>\*</sup>. Connection map for compounds (CMC): a server for combinatorial drug toxicity and efficacy analysis. **Journal of chemical information and modeling**, **2016**; 56(9), 1615–1621. DOI: https://doi.org/10.1021/acs.jcim.6b00397.

1. Ruixin Zhu<sup>†</sup>, Susan S Baker, Cynthia A Moylan, Manal F Abdelmalek, Cynthia D Guy, Fausto Zamboni, **Dingfeng Wu**, Weili Lin, WenshengLiu, Robert D Baker, Sugantha Govindarajan,Zhiwei Cao, Patrizia Farci, Anna Mae Diehl, and Lixin Zhu<sup>\*</sup>. Systematic transcriptome analysis reveals elevated expression of alcohol-metabolizing genes in NAFLD livers. **The Journal of pathology**, **2016**; 238(4), 531–542. DOI: https://doi.org/10.1002/path.4650.

1. Yi Sun<sup>†</sup>, Zhen Sheng<sup>†</sup>, Chao Ma, Kailin Tang, RuixinZhu, Zhuanbin Wu, Ruling Shen, Jun Feng, **Dingfeng Wu**, DanyiHuang, Dandan Huang, Jian Fei<sup>\*</sup>, Qi Liu<sup>\*</sup>, and Zhiwei Cao<sup>\*</sup>. Combining genomic and network characteristics for extended capability in predicting synergistic drugs for cancer. **Nature communications**, **2015**; 6, 8481. DOI: https://doi.org/10.1038/ncomms9481.

1. **Dingfeng Wu**<sup>†</sup>, Tianyi Qiu<sup>†</sup>, Qingchen Zhang, Hong Kang, Shaohua Yuan, Lixin Zhu<sup>\*</sup>, and Ruixin Zhu<sup>\*</sup>. Systematic toxicity mechanism analysis of proton pump inhibitors: an in silico study. **Chemical research in toxicology**, **2015**; 28(3), 419–430. DOI: https://doi.org/10.1021/tx5003782.

1. Tianyi Qiu<sup>†</sup>, Han Xiao, Qingchen Zhang, Jingxuan Qiu, Yiyan Yang, Dingfeng Wu, Zhiwei Cao<sup>\*</sup>, and Ruixin Zhu<sup>\*</sup>. Proteochemometric modeling of the antigen-antibody interaction: new fingerprints for antigen, antibody and epitope-paratope interaction. **PloS one**, **2015**; 10(4), e0122416. DOI: https://doi.org/10.1371/journal.pone.0122416.

1. Tao Qi<sup>†</sup>, Tianyi Qiu<sup>†</sup>, Qingchen Zhang, Kailin Tang, Yangyang Fan, Jingxuan Qiu, **Dingfeng Wu**, Wei Zhang, Yanan Chen, Jun Gao, Ruixin Zhu<sup>\*</sup>, and Zhiwei Cao<sup>\*</sup>. SEPPA 2.0--more refined server to predict spatial epitope considering species of immune host and subcellular localization of protein antigen. **Nucleic acids research**, **2014**; 42(Web Server issue), W59–W63. DOI: https://doi.org/10.1093/nar/gku395.

1. Jun Gao<sup>†</sup>, Qi Huang<sup>†</sup>, **Dingfeng Wu**, Qingchen Zhang, Yida Zhang, Tian Chen, Qi Liu, Ruixin Zhu, Zhiwei Cao<sup>\*</sup>, and Yuan He<sup>\*</sup>. Study on human GPCR-inhibitor interactions by proteochemometric modeling. **Gene**, **2013**; 518(1), 124–131. DOI: https://doi.org/10.1016/j.gene.2012.11.061.

1. **Dingfeng Wu**<sup>†</sup>, Qi Huang<sup>†</sup>, Yida Zhang, Qingchen Zhang, Qi Liu, Jun Gao, Zhiwei Cao<sup>\*</sup>, and RuixinZhu<sup>\*</sup>. Screening of selective histone deacetylase inhibitors by proteochemometricmodeling. **BMC bioinformatics**, **2012**; 13, 212. DOI: https://doi.org/10.1186/1471-2105-13-212.

**Academic Conference**

1. **Na Jiao**, Fusobacterium nucleatum, a reproducible microbial marker for CRC prescreening, Experimental Biology (EB 2021), Oral presentation. Online.

1. **Na Jiao**, Gut microbiota contributes to promoting secondary bile acid biosynthesis in NAFLD pathogenesis, Digestive Diseases Week (DDW 2019), Oral presentation. San Diego, CA.

1. **Na Jiao**, Suppressed hepatic bile acid signaling despite elevated production of primary and secondary bile acids in NAFLD, Digestive Diseases Week (DDW 2017), Oral presentation. Chicago, IL.

1. **Na Jiao**, Susan S. Baker, Colleen A. Nugent, Maria Tsompana, Liting Cai, Yong Wang, Michael J. Buck, Robert J. Genco, Robert D. Baker, Ruixin Zhu, Lixin Zhu, High-fat diet increases Clostridium clusters XIVa in obese rodents: a meta-analysis. Digestive Diseases Week (DDW 2017), Poster, Abstract #2674202, Chicago, IL.

1. Wei Xie, **Na Jiao**, Cenling Ma, Tommy. J. Phelps, Ruixin Zhu, Chuanlun Zhang, The Response of Archaeal Species to Seasonal Variables in a Subtropical Aerated Soil. Goldschmidt 2017, Poster, Abstract # gold2017: abs: 2017003171, Paris.

1. **Na Jiao**. Amplicon analysis and its application in disease. Symposium on Flora Analysis 2019. Oral presentation. Guangzhou, China


**Community Involvement**

1. **Frontiers in Microbiology** (IF=6.06). Review editor (2019-)

1. **Frontiers in Pharmacology** (IF=5.98). Review editor (2019-)
